Rakefile

```diff
@@ -20,3 +20,7 @@ end
 task :jekyll do
   system "bundle exec jekyll build"
 end
+
+task :server do
```

Comment:
I suppose you wanted to name the task "serve", not "server"?

Reply:
I'm not picky :) I meant to write "server" like in Rails. Do you have a preference?